src/ExportMap.js

```diff
@@ -579,7 +579,7 @@ ExportMap.parse = function (path, content, context) {
     }).digest('hex');
     let tsConfig = tsConfigCache.get(cacheKey);
     if (typeof tsConfig === 'undefined') {
-      tsConfig = readTsConfig(context);
+      tsConfig = readTsConfig();
```

Comment:
in this case, the bug is that `readTsConfig` *isn't* defining a `context` argument - so this line should be reverted, and line 550 fixed.

Reply:
Thanks, updated accordingly in https://github.com/import-js/eslint-plugin-import/pull/2589/commits/c7dc38bb5663db9e2c2e6693102255c0790f96a9